app/src/org/commcare/utils/MediaUtil.java

```diff
@@ -370,21 +372,83 @@ private static Bitmap scaleDownToTargetOrContainer(String imageFilepath,
         int approximateScaleDownFactor = getApproxScaleDownFactor(newWidth, originalWidth);
         Bitmap b = inflateImageSafe(imageFilepath, approximateScaleDownFactor).first;
 
+        int orientation = ExifInterface.ORIENTATION_NORMAL;
+        if (!imageFilepath.endsWith("png")) {

```

Comment:
I don't think we can practically maintain a list of what image formats have Exif data vs not and think our strategy here should be to try and get exif data safely for all file formats, but if that's not available simply don't do any exif manipulations and have the core functionality to display the image work without any crashes.